lib/msal-common/src/client/BaseClient.ts

```diff
@@ -43,7 +42,7 @@ export abstract class BaseClient {
     protected account: Account;
 
     // Default authority object
-    protected defaultAuthorityInstance: Authority;
+    protected authority: Authority;
```

Comment:
I don't think we should change the name of this field, it is still the defaultAuthorityInstance. Calling it just authority may lead to confusion down the road and makes the code a little less readable.

Reply:
This is a bit different than the way it was before. It made sense that this was defaultAuthorityInstance before, as its value was either the authority set on the application object or if none passed in, then the common endpoint. When the user made an acquireToken or Login call, if they didn't "override" the authority in the call, then we defaulted to this value. 

Now that is all done in either node or browser, and this is just the final authority that will be used for that request. We don't differentiate between a default authority and a overriden value.

Reply:
In the context of node + common I agree, but in the context of just msal-common, this is still for all intents and purposes the default authority that is used if there is none passed in the request correct?

Reply:
Originally this PR only moved authority generation for node. Yesterday I made similar changes for browser now that configuration has been aligned. This distinction is only applicable for SPAClient. I changed it back to defaultAuthority to reflect that difference, although it doesn't really apply for anything else.